oci/oci.go

```diff
@@ -486,6 +486,9 @@ func (r *Runtime) UpdateStatus(c *Container) error {
 	defer c.opLock.Unlock()
 	out, err := exec.Command(r.Path(c), "state", c.name).CombinedOutput()
 	if err != nil {
+		if strings.Contains(string(out), "does not exist") {
```

Comment:
actually, you already said what I've proposed here https://github.com/kubernetes-incubator/cri-o/pull/505#discussion_r115986596 so yeah, will fix this.